cdap-common/bin/common.sh

```diff
@@ -267,6 +267,10 @@ cdap_set_hive_classpath() {
     # classpath from it and export it to use it in the launch command
     if [ -n "${HIVE_HOME}" -a -n "${HIVE_CONF_DIR}" -a -n "${HADOOP_CONF_DIR}" ]; then
       EXPLORE_CONF_FILES=$(ls -1dF ${HIVE_CONF_DIR}/* ${HADOOP_CONF_DIR}/* | sed -e '/\/$/d' | tr '\n' ':')
+      if [ -n "${TEZ_HOME}" ]; then
+        # tez-site.xml also need to be passed to explore service
+        EXPLORE_CONF_FILES=${EXPLORE_CONF_FILES}:${TEZ_HOME}/conf/tez-site.xml:
```

Comment:
We need to load the jars in java code similar to spark. Seems like jars added in the HIVE_CLASSPATH are only used for re-writing `HiveAuthFactory` while loaded jars using java are explicitly passed to container classpath.